docker/libnetwork
# ipset layer for docker

**- What I did**
An ipset layer for docker. Currently is only able to do create (with validation), save, delete, flush, destroy. A lot more left to do. This is to get a go ahead or feedback as to whether I can continue on.

**- How I did it**
Taken in parameters, verify validity, build execution output

**- How to verify it**
There's a sample program in the README.md which can be used to test it. you can change ipset to any script that will just echo out arguments passed to see what has been passed as arguments also.



seeking feedback, work in progress prototype
Related issues this can be used to solve: https://github.com/docker/docker/issues/15830
